.gitignore

```diff
@@ -1,5 +1,10 @@
 node_modules
 test-bundles
 output
+synthea
+connectathon
+.cqf-ruler
+.seed-cqf-ruler
+.setup-cqf-ruper
```

Comment:
Minor typo in `ruler` here, but otherwise looks good.

Reply:
good catch!